web/views/utils.py

```diff
@@ -18,6 +18,13 @@ def is_data_steward(user):
         raise PermissionDenied
 
 
+def can_publish(user):
+    if user.is_superuser or user.is_part_of(Group.objects.get(name=Groups.DATA_STEWARD.value)):
```

Comment:
I would suggest replacing this condition by `if user.can_publish()`.

This would avoid future issues if we have to change the `can_publish` method in User class but forget that modification here.

Reply:
done